Localization patch. jenkins stable-2.492.

YoLoveLife opened the pull request:
jenkins version changes causing field renaming leading to localization failures, correcting field names, adding some new localized content.
role-strategy-plugin: Partially missing Chinese localization fields.

<!--
您好，非常高兴收到您的 PR。为了让我们可以更好地协作，请注意下面的事项：

* 请保留当前 PR 模板中的所有内容，包括注释，如果有需要添加的内容，直接添加即可
* 请您每次使用不同的分支来提交 PR，避免使用您的 master 来修改并提交
* 请您尽量给出一个可读性比较强的 PR 标题，这样方便维护者以及其他贡献者 review 以及检索

-->

### Submitter checklist

- [x] [Know translation specification](https://github.com/jenkins-zh/translation-spec/blob/master/specification.md)

Here is [an online tool](https://native2ascii.net/) which can help you to do the review work.

### Desired reviewers

@jenkinsci/chinese-localization-sig

YoLoveLife committed:
jenkins version changes causing field renaming leading to localization failures, correcting field names, adding some new localized content.

role-strategy-plugin: Partially missing Chinese localization fields.

YoLoveLife committed:
Merge pull request #1 from jenkinsci/master

fix

LinuxSuRen commented on core/src/main/resources/hudson/PluginManager/installed_zh_CN.properties:
```diff
@@ -34,3 +34,38 @@ Update\ Center=\u66F4\u65B0\u4E2D\u5FC3
 Filter=\u8FC7\u6EE4
+
+# jenkins branch stable-2.492,introduction of new fields
+requires.restart=此 Jenkins 執行個體需要重新啟動。強烈建議您不要在此時變更外掛的狀態。請在操作前重新啟動 Jenkins。
```
我们这个插件是简体中文的，麻烦您更新下？

YoLoveLife replied:
好的。这部分内容因为繁体汉化的字段更全一点，想着沿用一下，这部分我再修正一下

YoLoveLife committed:
修改由原Jenkins源码引入的繁体汉化文本
Modify the traditional Chinese localization text introduced by the original Jenkins source code.

YoLoveLife committed:
Merge branch '20250319' of github.com:YoLoveLife/localization-zh-cn-plugin into 20250319

YoLoveLife commented:
Already modified, please review.
